controller_manager/CMakeLists.txt

```diff
@@ -2,65 +2,96 @@ cmake_minimum_required(VERSION 2.8.3)
 project(controller_manager)
 
 # Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED COMPONENTS controller_interface controller_manager_msgs hardware_interface pluginlib)
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  controller_manager_msgs
+  hardware_interface
+  pluginlib
+  roscpp
+)
 
+# Include Boost.Thread
 find_package(Boost REQUIRED COMPONENTS thread)
 
-include_directories(include ${Boost_INCLUDE_DIR} ${catkin_INCLUDE_DIRS})
+catkin_python_setup()
 
-# Declare catkin package
+# Declare a catkin package
 catkin_package(
-  DEPENDS Boost
-  CATKIN_DEPENDS controller_interface controller_manager_msgs hardware_interface pluginlib
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME}
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    ${PROJECT_NAME}
+  CATKIN_DEPENDS
+    controller_interface
+    controller_manager_msgs
+    hardware_interface
+    pluginlib
+    roscpp
+  DEPENDS
+    Boost
 )
 
+
+###########
+## Build ##
+###########
+
+# Specify header include paths
+include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIR})
```

Comment:
From the docs on `FindBoost.cmake` ([these](https://cmake.org/cmake/help/v3.0/module/FindBoost.html)) I get the impression `Boost_INCLUDE_DIRS` should be used here. See the examples near the bottom of the page.

Reply:
Good catch!

Reply:
Fixed in def3541